src/main/java/org/glassfish/concurro/cdi/ConcurrencyManagedCDIBeans.java

```diff
@@ -37,7 +37,7 @@ public enum Type {
     /**
      * Annotation scanner stores this instance in JNDI under JNDI_NAME name.
      */
-    public static final String JDNI_NAME = "java:app/concurrent/__ConcurrencyManagedCDIBeans";
+    public static final String JNDI_NAME = "java:app/concurrent/__ConcurrencyManagedCDIBeans";
```

Comment:
```java
    public static final String JNDI_NAME = "java:app/concurrent/__ConcurrencyManagedCDIBeans";
    @Deprecated public static final String JDNI_NAME = JNDI_NAME;
```
?

Reply:
Perhaps not needed, as the constant seems to appear after 3.0.0, and so far there was no 3.1.0 finally released yet.